packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -53,6 +64,55 @@ const getInputElementForType = (type: FormControlProps['type']) => {
   return CustomInputs[customInput] || Input;
 };
 
+function useFormTextAnimation() {
+  const prefersReducedMotion = usePrefersReducedMotion();
+
+  const getFormTextAnimation = useCallback(
+    (enterAnimation: boolean): ThemableCssProp => {
+      if (prefersReducedMotion) {
+        return {
+          animation: 'none',
+        };
+      }
+      return t => ({
+        animation: `${enterAnimation ? animations.inAnimation : animations.outAnimation} 300ms ${
```

Comment:
Can't we use `t.transitionDuration.slower` here?

Reply:
I hard coded for the following reason:

Inside FormControl, I use `useDelayUnmount` and I pass a specific value of 200ms, I found out through trial and error that the sweet point for the animation duration needs to be 100ms for than the input value in `useDelayUnmount`

Using this `t.transitionDuration.slower` means that a developer can override these values, causing the animation to break

Reply:
- I could not find a way to override them from the appearance prop.
- t.transitionDuration.slower is currently set to 280ms, the animation still looks pretty ok with this.

I am switching to `t.transitionDuration.slower`